infra/runtests.py

```diff
@@ -49,6 +49,15 @@ def reload_module(mod):
             delattr(mod, attr)
     importlib.reload(mod)
 
+def import_as(file_path, module_name):
```

Comment:
```suggestion
def import_file_as(file_path, module_name):
```

Also add a comment explaining in English what this method does.